cmd/common/image/build.go

```diff
@@ -217,6 +200,138 @@ func runBuild(opts *cmdOpts) error {
 	return nil
 }
 
+func buildLocal(opts *cmdOpts, conf *buildFile, output string) error {
+	makefilePath := filepath.Join(output, "Makefile")
+	if err := os.WriteFile(makefilePath, makefile, 0o644); err != nil {
+		return fmt.Errorf("writing Makefile: %w", err)
+	}
+
+	buildCmd := exec.Command(
+		"make", "-f", makefilePath, "-j", "2",
```

Comment:
I guess because we have 2 architectures :D For `nproc` we need to query that via `exec.Command` again.
Using [`runtime.NumCPU()`](https://pkg.go.dev/runtime#NumCPU) would be better in this case then

Reply:
I added 2 because that's the number of architectures we have. Changed it to use [runtime.NumCPU()](https://pkg.go.dev/runtime#NumCPU).